src/backend/utils/html/sanitize.js

```diff
@@ -49,12 +52,21 @@ module.exports = function (dirty) {
     },
     allowedIframeHostnames: [
       'www.youtube.com',
-      'www.twitch.tv',
+      'player.twitch.tv',
       'player.vimeo.com',
       'giphy.com',
       'cdn.embedly.com',
       'open.spotify.com',
       'medium.com',
     ],
+    transformTag: {
```

Comment:
OK, this is really cool, nice work @HyperTHD, I didn't know we could do this.

Reading their docs, I think you want something like this (untested code!):

```js
transformTags: {
  iframe: (tagName, attribs) => {
    // Swap the parent param for Twitch iframe URLs. For example, use a new host for dev.to in:
    // https://player.twitch.tv/?video=1303803789&parent=dev.to&autoplay=false
    const swapParent = (parent, host) => {
      try {
        const urlObj = new URL(attribs.src);
        // Only bother if this is a Twitch embed iframe
        if(urlObj.host === 'player.twitch.tv' && urlObj.searchParams.get('parent')) {
          urlObj.searchParams.set('parent', host);
        }
        return urlObj.href;
      } catch {
        // Return the original untouched
        return url;
      }
    };

    // Merge all the other attributes, but fix src
    return {
      tagName,
      attribs: {
        ...attribs,
        src: swapParent(attribs.src, new URL(WEB_URL).host),
      },
    },
  }
}
```

Reply:
Worth a shot, hard to test locally since I'd have to re-index.